app/models/user.rb

```diff
@@ -5,4 +5,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :rememberable, :trackable, :validatable
 
   devise :omniauthable, omniauth_providers: [:bnet]
+
+  scope :anonymous, -> { find_by_email "anonymous@ghost.com" }
```

Comment:
Does this make sense as a scope? My impression of scopes is that they're chainable, but this one isn't:

```
>> User.find_by_email('hey@you.com').where(id: 3)
NoMethodError: undefined method `where' for nil:NilClass
```

What about making it a class method instead?